### PR TITLE
Prf lsh svd

### PR DESCRIPTION
**Pull request by Kunal30**

### CODE REVIEW

**Comment by AdityaGovardhan** [comment]
Rebase!

File: src/backend/utils.py
Comment (AdityaGovardhan) [inline src/backend/utils.py:9]: @vibnimit

File: src/backend/utils.py
Comment (AdityaGovardhan) [inline src/backend/utils.py:51]: this function is already present above at line 30

Comment (Kunal30) [reply src/backend/utils.py:51]: dafaq mere waale me it was missing.... had to write it again..... looks like ur changes came

**Comment by AdityaGovardhan** [comment]
Added comments. Looks good to integrate. Will approve after @skuma149 and @vibnimit approval

**Review by skuma149 — DISMISSED** [review]
Sorting K images is being called from k similar images

**Review by AdityaGovardhan — APPROVED** [review]
Looks good to merge

File: src/backend/relevance_feedback.py
Comment (skuma149) [inline src/backend/relevance_feedback.py:131]: Need to provide k and l value while initializing

**Comment by Kunal30** [comment]
These are just pseudo code comments don’t worry

**Review by skuma149 — APPROVED** [review]

**Pull request merged**
